mongodbatlas/resource_mongodbatlas_network_container_test.go

```diff
@@ -245,7 +245,7 @@ func testAccMongoDBAtlasNetworkContainerConfigAzure(projectID, cidrBlock, provid
 			project_id   		 = "%s"
 			atlas_cidr_block = "%s"
 			provider_name		 = "%s"
-			region			     = "US_EAST_2"
+			region			     = "US_EAST"
```

Comment:
Why the change? (both are Azure regions https://docs.atlas.mongodb.com/reference/microsoft-azure/)

Reply:
Because it fails with US_EAST_2 and I don't know how to delete it with mongodb ui, it doesn't appear that container the same can be said for GCP, that's why I changed it so it can pass, do you want me to change it back?

Reply:
You can't delete containers in the MongoDB UI because we don't show it there, only via API.  I'm not sure I follow on why it might be failing, what's the actual error?  Might be showing us a real problem.